runtime-builder/build.sh

```diff
@@ -14,18 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-usage() { echo "Usage: $0 [project_id] [go_version]"; exit 1; }
+usage() { echo "Usage: $0 GO_VERSION [DEBIAN_TAG]"; exit 1; }
 
 set -e
 
-export PROJECT="$1"
-export GO_VERSION="$2"
+export GO_VERSION="$1"
 export BUILD_TAG="$GO_VERSION"-`date +%Y-%m-%d_%H_%M`
```

Comment:
Not sure if there's an official pattern that we need to follow.  If not, I prefer a shorter form +%Y%m%d_%H%M